ezproxylookup/views.py

```diff
@@ -1,58 +1,47 @@
-from flask import Flask, request, jsonify, render_template, url_for, redirect
-import json
-from dotenv import load_dotenv
-import boto3
-
-# flask command does this on its own. we need to do it explicity for gunicorn
-load_dotenv('.env')
-
-app = Flask(__name__)
-
-CONFIG_FILE = 'config.json'

```

Comment:
Did you remove this on purpose? Just curious. I found it useful :)

Reply:
I did. It seemed like a hacky way to do what its doing. I am planning to start writing some tests which I think would have made this redundant.

Reply:
Yeah that makes sense, and is a better solution 👍